app/assets/javascripts/angular/controller/forms/modalChangeNameCtrl.js.erb

```diff
@@ -0,0 +1,26 @@
+angular.module('formidable').controller('ModalChangeNameCtrl', function ($scope, $modalInstance, $http, $location, $routeParams, $rootScope) {
+
+  url = "<%= ENV['URL'] %>" || 'http://localhost:3000';
+  
+
+  $scope.changeFormName = function(newName) {
+    $scope.form = {
+      name: newName
+    };
+    $http.put(url + '/forms/' + $routeParams.id ,$scope.form).success(function(response) {
+       
```

Comment:
you guys should notifiy the user when there is an action done, for example, you can use [bootstrap alerts](http://getbootstrap.com/javascript/#alerts) with an auto-dismiss after 5 secs?